lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -198,15 +198,16 @@ void InterpreterFunction::fwdConvolutionInst_I8Impl(
             }
 
             // Scale the bias to match the scale of the matrix multiplication.
-            int32_t B = std::round(float(biasW.at({d}) - biasOffset) *
+            int32_t B = std::round(float(biasW.unsafe_at({d}) - biasOffset) *
```

Comment:
Just a personal opinion: it feels strange to have unsafe_at here and not in the other places.

If we want to selectively enable faster access with a good granularity, I guess we could have something like:
::at(..., bool skipAssert = false) {
assert(skipAssert || (<regularAssertCode>))
}

Then we can instrument each access with macros, like USE_FAST_CONV.

Just an idea though. I expect this to be relatively ugly while not completely grasping the intent here x).

Reply:
Tbh, I kind of wanted this call to be jarring from an API perspective.  No one should be using this except for a well understood place like convolution, where we already have type checks in place to make sure e.g. your filter/input/output dimensions match.